neuralmonkey/vocabulary.py

```diff
@@ -602,12 +602,16 @@ def save_wordlist(self, path: str, overwrite: bool = False,
 
                 output_file.write("\n")
 
-    def log_sample(self, size: int = 5):
+    def log_sample(self, size: int = 5) -> None:
         """Log a sample of the vocabulary.
 
         Arguments:
             size: How many sample words to log.
         """
-        log("Sample of the vocabulary: {}"
-            .format([self.index_to_word[i]
-                     for i in np.random.randint(0, len(self), size)]))
+        if size > len(self):
+            log("Vocabulary: {}".format(
+                [self.index_to_word[i] for i in range(0, len(self))]))
```

Comment:
Je tam potřeba ta složená závorka? Nešlo by tam napsat prostě generátor bez závorek?

Reply:
když jí tam nedáš, tak to řekne
```
Sample of the vocabulary: <generator object Vocabulary.log_sample.<locals>  (...)
```

Reply:
OK. Ale `self.index_to_word` je list, tak ho tam stačí ronvou napsat.